Features/tijdreizen.feature

```diff
@@ -0,0 +1,112 @@
+    En dat met de kennis van toen geldig is op het moment van bevragen
+  
+  Scenario: geldigOp wel en beschikbaarOp niet opgegeven in request
+    Als bij een endpoint geldigOp en beschikbaarOp ondersteund worden
+    En geldigOp is wel opgegeven in het request
+    En beschikbaarOp is niet opgegeven in het request
+    Dan bevat de response het voorkomen van een object dat op dit moment geregistreerd staat
+    En dat met de kennis van dit moment, geldig was op het moment geldigOp
+  
+  Scenario: geldigOp en beschikbaarOp opgegeven in request
+    Als bij een endpoint geldigOp en beschikbaarOp ondersteund worden
+    En geldigOp is wel opgegeven in het request
+    En beschikbaarOp is wel opgegeven in het request
+    Dan bevat de response het voorkomen van een object dat geregistreerd stond op het opgegeven moment beschikbaarOp
+    En dat met de kennis van toen, geldig was op het moment geldigOp
+  
+  Scenario: endpoint ondersteunt query parameters geldigOp en beschikbaarOp niet
+    Als bij een endpoint geldigOp en beschikbaarOp niet wordt ondersteund
+    En in een request wordt geldigOp en/of beschikbaarOp opgegeven
+    Dan treedt de volgende foutmelding op:
+    | Foutsituatie                        | status | voorbeeld title                                            | code              |
+    | Niet toegestane parametercombinatie | 400    | De combinatie van opgegeven parameters is niet toegestaan. | unsupportedCombi  |
+
+  Scenario: query parameter huidig in request, status object voorkomen is geen eindstatus
+    Als bij een endpoint geldigOp, beschikbaarOp en huidig worden ondersteund
+    En in een request wordt geen geldigOp opgegeven
+    En in de request wordt geen beschikbaarOp opgegeven
+    En in de request wordt huidig opgegeven
+    En de status van het betreffende object voorkomen is geen eindstatus
+    Dan bevat de response het actuele voorkomen van een object
+
+  Scenario: query parameter huidig in request, status object voorkomen is eindstatus
+    Als bij een endpoint geldigOp, beschikbaarOp en huidig worden ondersteund
+    En in een request wordt geen geldigOp opgegeven
+    En in de request wordt geen beschikbaarOp opgegeven
+    En in de request wordt huidig opgegeven
+    En de status van het betreffende object voorkomen is geen eindstatus
+    Dan bevat de response geen resultaat
+
+  Scenario: query parameter geldigOp, beschikbaarOp en huidig in request
+    Als bij een endpoint geldigOp, beschikbaarOp en huidig worden ondersteund
+    En in een request wordt geldigOp opgegeven
+    En in de request wordt beschikbaarOp opgegeven
+    En in de request wordt huidig opgegeven
+    Dan treedt de volgende foutmelding op:
+    | Foutsituatie                        | status | voorbeeld title                                            | code              |
+    | Niet toegestane parametercombinatie | 400    | De combinatie van opgegeven parameters is niet toegestaan. | unsupportedCombi  |
```

Comment:
Bij de bag-api hanteren we de code `unsupportedCombination`, zie ook https://github.com/lvbag/BAG-API/blob/master/Features/foutafhandeling.feature#L17